src/Encamina.Enmarcha.Email.Abstractions/EmailAttachmentSpecification.cs

```diff
@@ -10,12 +10,12 @@ public class EmailAttachmentSpecification
     /// <summary>
     /// Gets the file name of the attachment in this specification.
     /// </summary>
-    public string FileName { get; init; }
+    public string? FileName { get; init; }
```

Comment:
I'm not sure if the `FileName `property should be nullable 🤔 From what I can see, this property is used to construct the attachments, and if it were null, it would generate an exception (https://github.com/jstedfast/MimeKit/blob/245f14b657bce378f96efd7ccc2428f2669b1812/MimeKit/AttachmentCollection.cs#L366).